solidus_dummy_extension/solidus_dummy_extension.gemspec

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require_relative 'lib/solidus_dummy_extension/version'
+
+Gem::Specification.new do |spec|
+  spec.name = 'solidus_dummy_extension'
```

Comment:
I found that it's simpler to test the orb against a static (versus a dynamically-generated) dummy extension. Furthermore, it would allow us to test the dummy extension locally.